ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -64,6 +67,7 @@ - (instancetype)initWithUIManager:(RCTUIManager *)uiManager
       [_targetKeys addObject:[NSString stringWithFormat:@"target%@", [key capitalizedString]]];
       [_currentKeys addObject:[NSString stringWithFormat:@"current%@", [key capitalizedString]]];
     }
+    _sharedTransitionManager = [[REASharedTransitionManager new] init:self];
```

Comment:
I think that the objc convention is for `new` to be the same as `[[x alloc] init]`, so this looks like initialising the same object twice.
Not sure it breaks anything here, but it may be better to replace the `new` here with `alloc`.